app/models/container_deployment.rb

```diff
@@ -1,4 +1,5 @@
 class ContainerDeployment < ApplicationRecord
+  include ContainerDeploymentMixin
```

Comment:
This is only going to be used one place, correct?  If so it should not be a mixin, use `include_concern` instead.

For example: ems_event.rb contains `include_concern 'Automate'`
https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_event.rb#L2

then the automate module is defined here: https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_event/automate.rb#L1